### PR TITLE
Ability to use variable databags per environment / role / etc

### PR DESCRIPTION
**Pull request by snkutlu**
With attribute settings, we will be able to use different acls, hosts and urls per each environment / role / node / etc.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Installs/configures squid as a simple caching proxy'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.5.2'
+version          '0.5.2001'
```
Comment (juliandunn) [inline metadata.rb]: Please don't modify the version #.

**Comment by juliandunn** [comment]
Seems legit, need a rebase.

**Comment by snkutlu** [comment]
Hey @juliandunn 

I rebased it.

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by juliandunn** [comment]
Thanks @snkutlu. Can you review the CLA message above and then I can take this.

**Comment by snkutlu** [comment]
@juliandunn, I am already listed in contributer list of Schuberg Philis B.V. which has CLA signed. I guess this will be enough ;)

**Comment by juliandunn** [comment]
ah yeah, I'm sure it is just because whatever email address is on the individual commits is not the same one as attached to your CLA... but it's cool.

**Comment by juliandunn** [comment]
squashed and merged in 496aaabd5b4809be272957157638703a610af10e

**Pull request closed without merging**